### PR TITLE
Update the projected values in Extend

### PR DESCRIPTION
**Pull request by kapetanios55**
There is currently an issue with this analytic rule because the "threshold" value is not projected. The issue is within the Alert description tab

   Required items, please complete
   
   Change(s):
   - added the threshold in the extended properties to mitigate an issue with the Alert Description tab - This allows the threshold to be projected as a value in the title

   Reason for Change(s):
   - Error, that prevents the rule from being created - "Column name threshold does not exist. Please check the query results and try again"

   Version Updated:
   - Yes
   - 1.0.1

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

File: Detections/ASimNetworkSession/ExcessiveHTTPFailuresFromSource.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ description: |
 severity: Medium
 tags:
   - ParentAlert: https://github.com/Azure/Azure-Sentinel/blob/master/Detections/SophosXGFirewall/ExcessiveAmountofDeniedConnectionsfromASingleSource.yaml
-    version: 1.0.0
+    version: 1.0.1
```
Comment (oshezaf) [inline Detections/ASimNetworkSession/ExcessiveHTTPFailuresFromSource.yaml:9]: Please update the version number in line 63, not this one.

**Review by oshezaf — CHANGES_REQUESTED** [review]

File: Detections/ASimNetworkSession/ExcessiveHTTPFailuresFromSource.yaml (hunk 1 above)
Comment (kapetanios55) [reply Detections/ASimNetworkSession/ExcessiveHTTPFailuresFromSource.yaml]: Done

Comment (oshezaf) [reply Detections/ASimNetworkSession/ExcessiveHTTPFailuresFromSource.yaml]: Did you update the version in line 63 from 1.2.1 to 1.2.2? I can't see it. If you updated on your desktop, you need to push the commit. I often forget to do that :-)

**Comment by v-atulyadav** [comment]
Hi @oshezaf, @kapetanios55 has commit the change you requested please check it and provide your feedback accordingly. Thanks.

**Review by oshezaf — APPROVED** [review]

**Pull request merged**